app/extensions/brave/locales/en-US/extensions.properties

```diff
@@ -29,3 +29,4 @@ torrent= Torrent Viewer
 torrentDesc=Uses WebTorrent to display torrents directly in the browser. Supports torrent files and magnet links.
 vimium= Vimium
 vimiumDesc=
+honeyDesc=Automatically find and apply coupon codes when you shop online!
```

Comment:
@bsclifton `honey=Honey` was originally suggested here. @cezaraugusto, please see the discussion [here](https://github.com/brave/browser-laptop/pull/8120#pullrequestreview-35667733) regarding this change.